fava/static/sass/_journal-table.scss

```diff
@@ -244,6 +244,20 @@
     &.leg-pending { background-color: $color-journalentry-postingwarning; }
   }
 
+  .metadata-indicator {
+    font-size: 10px;
```

Comment:
Which specific error do you mean? I only get warnings from `sass-lint`, not errors.